lib/connect.c

```diff
@@ -1056,8 +1056,8 @@ CURLcode Curl_is_connected(struct Curl_easy *data,
       hostname = conn->http_proxy.host.name;
     else
 #endif
-      if(conn->bits.conn_to_host)
-        hostname = conn->conn_to_host.name;
+    if(conn->bits.conn_to_host)
+      hostname = conn->conn_to_host.name;
```

Comment:
why?

Reply:
Because the formatting here is broken

Reply:
in your opinion then perhaps, but not in mine. The `if` is intended based on the `else` on the line before.